### DISCUSSION TITLE
WARNING : major bug found in 3.8.0

### DISCUSSION THREAD

**Discussion by jmcollin78** [jmcollin78/versatile_thermostat#153] [Announcements]
**Please install 3.8.1 imperatively.**

The bug can lead a heater On or Off wrongly.

**Comment by Ba-pt0u** [comment]
I confirm ! I wake up at 2AM because it was too hot in the bedroom 😄
Thank you for the quick patch, i opened my phone, and apply the hotfix and everything seems to be back to normal. 👍
